mods/mesecons/mesecons_luacontroller/init.lua

```diff
@@ -167,7 +167,7 @@ local function burn_controller(pos)
 	local node = minetest.get_node(pos)
 	node.name = BASENAME.."_burnt"
 	minetest.swap_node(pos, node)
-	--minetest.get_meta(pos):set_string("lc_memory", "");
+	minetest.get_meta(pos):set_string("lc_memory", "");
```

Comment:
I commented this out because player's were complaining of loosing data on overheated luacontrollers.